src/vivarium_public_health/metrics/stratification.py

```diff
@@ -44,14 +44,16 @@ def register_stratifications(self, builder: Builder) -> None:
             is_vectorized=True,
             requires_columns=["event_time"],
         )
+        # TODO [MIC-3892]: simulants occasionally have entrance year of start_year-1 if the start time minus step size
+        # TODO: lands in the previous year. possible solution detailed in ticket
```

Comment:
You shouldn't need to put TODO here, just indent following lines by 2 spaces. Should highlight properly in PyCharm.

```suggestion
        #   lands in the previous year. possible solution detailed in ticket
```

Reply:
Haha you knew exactly why I did that but your way worked!